README.md

```diff
@@ -22,11 +22,23 @@ Pre-build images are available on [DockerHub](https://hub.docker.com/r/existdb/e
 To download the image run:
 ```bash
 docker pull existdb/existdb:latest
-docker run -it -d -p 8080:8080 -p 8443:8443 existdb/existdb:latest
 ```
 
-You can now access eXist via [localhost:8080](localhost:8080) in your browser.
-Try to stick with matching internal and external port assignments, to avoid unnecessary reloads and connection issues.
+once the download is complete, you can run the image
+```bash
+docker run -it -d -p 8080:8080 -p 8443:8443 --name exist existdb/existdb:latest
+```
+
+What does this do?
```

Comment:
Should this be a heading or in bold or something?

Reply:
can do